include/nccl_ofi_msgbuff.h

```diff
@@ -43,7 +43,7 @@ extern "C" {
  */
```

Comment:
Similar to https://github.com/aws/aws-ofi-nccl/pull/561, this is not the solution we're looking for.